DataSetDocker/ReadMe.md

```diff
@@ -0,0 +1,22 @@
+# To build the image:
+
+From the directory containe Dockerfile launch:
+
+```sh
+docker build --tag dataset .
+```
+
+# To start docker: 
+
+```sh
+docker run -p 8888:8888 dataset
+```
+# To access the jupyter:
+
+Now you have the console with jupyter-lab server running.
+
+1. Copy the address with token from the console and put this line in you browser:
+
+``` 
+127.0.0.1:8888/?token=1708633ec3f24c945c8cc18ce0ca40f90d2ac94a0a55119f
```

Comment:
Where does this token come from? Is it always the same?

Reply:
No, it is generated by jupyter server